pkgs/by-name/sa/sarif-converter/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitLab,
+}:
+
+buildGoModule rec {
+  pname = "sarif-converter";
+  version = "0.9.4";
+
+  src = fetchFromGitLab {
+    owner = "ignis-build";
+    repo = "sarif-converter";
+    tag = "v${version}";
+    hash = "sha256-Q8L4C4mTJEfxASqrUAZlMoLx71/+nmP6mwW/vNFStTs=";
+  };
+
+  vendorHash = "sha256-vK+HhHlFWoWIrDEZzfRoqtJ3vKp0f4b8l8+LBlZuBJU=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
```

Comment:
consider adding versions so that versionCheckHook works like here?
https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/tr/trojan-go/package.nix#L23-L24